cmd/tailscale/cli/up.go

```diff
@@ -451,15 +455,37 @@ func runUp(ctx context.Context, args []string) error {
 		}
 		if url := n.BrowseToURL; url != nil && printAuthURL(*url) {
 			printed = true
-			fmt.Fprintf(Stderr, "\nTo authenticate, visit:\n\n\t%s\n\n", *url)
-			if upArgs.qr {
+			if upArgs.json {
+				type fields struct {
```

Comment:
if we're going to support this as an interface, it's worth making this a top-level type with docs on what the fields mean.

Reply:
Moved to top level